data/src/main/java/ca/bc/gov/data/datasource/remote/model/request/CommentRequest.kt

```diff
@@ -0,0 +1,11 @@
+package ca.bc.gov.data.datasource.remote.model.request
+
+/**
+ * @author: Created by Rashmi Bambhania on 18,April,2022
+ */
+data class CommentRequest(
+    val text: String,
+    val parentEntryId: String?,
+    val userProfileId: String?,
+    val entryTypeCode: String?
```

Comment:
Are we not going to use below fields as per the swagger 
"version": 0,
  "createdDateTime": "2022-04-22T08:08:29.027Z",
  "createdBy": "string",
  "updatedDateTime": "2022-04-22T08:08:29.027Z",
  "updatedBy": "string"

Reply:
Not required